docs/samples/Microsoft.ML.Samples/Microsoft.ML.Samples.csproj

```diff
@@ -5,6 +5,7 @@
     <OutputType>Exe</OutputType>
     <SignAssembly>false</SignAssembly>
     <PublicSign>false</PublicSign>
+    <RootNamespace>Samples</RootNamespace>
```

Comment:
Thought I might opportunistically do this, since our new policy per #3205 is that samples not have `Microsoft.ML` in their namespace, but *by default* we were still doing the "wrong" thing when we create a new file. I think it might be nice if we avoided the problem by default.